### PR TITLE
Update CM2 CoreOS

### PR DESCRIPTION
**Pull request by vlewisRH**
tracked via:
        https://issues.redhat.com/browse/CMP-260

### CODE REVIEW

File: coreos-4/policies/CM-Configuration_Management/component.yaml [hunk 1]
```diff
@@ -32,11 +32,33 @@
...
+        Ignition is meant to initialize systems, not change existing systems. After
+        a machine initializes and the kernel is running from the installed system, the
+        Machine Config Operator from the OpenShift Container Platform cluster completes
+        all future machine configuration.
 
+        Operators are a method of packaging, deploying, and managing a Kubernetes
+        application. Watching over the Kubernetes environment, using its current
+        state to make decisions in real time. The Machine Config Operator (MCO)
+        manages updates and configuration changes to essentially everything between
+        the kernel and kubelet.
```
Comment (shawndwells) [inline coreos-4/policies/CM-Configuration_Management/component.yaml]:
```suggestion
        the host operating system and kubelet.
```

Comment (shawndwells) [reply coreos-4/policies/CM-Configuration_Management/component.yaml]: @vlewisRH Would it still be accurate to say "host operating system" vs kernel?

File: coreos-4/policies/CM-Configuration_Management/component.yaml [hunk 2]
```diff
@@ -32,11 +32,33 @@
...
+
+        For additional information:
+        https://docs.openshift.com/container-platform/latest/architecture/architecture-rhcos.html#rhcos-about-ignition_architecture-rhcos
+        https://docs.openshift.com/container-platform/latest/operators/olm-what-operators-are.html
+        https://github.com/openshift/machine-config-operator
+
+        tracked via:
         https://issues.redhat.com/browse/CMP-260
 
+
```
Comment (shawndwells) [inline coreos-4/policies/CM-Configuration_Management/component.yaml]:
```suggestion

```

Comment (shawndwells) [reply coreos-4/policies/CM-Configuration_Management/component.yaml]: No longer need this issue tracker link.

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**